pkgs/tools/networking/ivpn/default.nix

```
@@ -69,7 +69,7 @@ builtins.mapAttrs (pname: attrs: buildGoModule (attrs // rec {
       substituteInPlace daemon/service/platform/platform_linux_release.go \
         --replace 'installDir := "/opt/ivpn"' "installDir := \"$out\"" \
         --replace 'obfsproxyStartScript = path.Join(installDir, "obfsproxy/obfs4proxy")' \
-        'obfsproxyStartScript = "${obfs4}/bin/obfs4proxy"' \
+        'obfsproxyStartScript = "${obfs4}/bin/lyrebird"' \
```

I think it would be better to change to:
```suggestion
        'obfsproxyStartScript = "${lib.getExe obfs4}"' \
```